core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -650,6 +646,23 @@ object ConsumerGroupCommand extends Logging {
       }
     }
 
+    private def checkOffsetRange(topicPartition: TopicPartition, shiftedOffset: Long) = {
+      val newOffset = getLogEndOffset(topicPartition) match {
```

Comment:
You're right. This should also apply to the `null`s returnedon `#prepareOffsetsToReset`, and instead return `#printUsageAndDie`. Do you agree?